dev-lua/tolua++/tolua++-1.0.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="An extension of toLua, a tool to integrate C/Cpp code with Lua"
+DESCRIPTION="tolua++ is an extension of toLua, a tool to integrate C/C++ code with \
+Lua. tolua++ includes new features oriented to c++, such as class templates."
+HOMEPAGE="https://github.com/LuaDist/toluapp"
+COPYRIGHT="2009 Ariel Manzur"
+LICENSE="MIT"
+REVISION="1"
+COMMIT_SHA="b34075b76835b778bb6b2ce0aa224afd9d182887"
+SOURCE_URI="https://github.com/LuaDist/toluapp/archive/$COMMIT_SHA.zip"
+CHECKSUM_SHA256="c69a99a4b0a2bb3decab09bd8d2473911419bef601c86bb6c00ff670f88b4bb1"
+SOURCE_DIR="toluapp-$COMMIT_SHA"
```

Comment:
Thanks @timfel for your PR!
Could you please make these neutral changes?
```diff
-COMMIT_SHA="b34075b76835b778bb6b2ce0aa224afd9d182887"
+srcGitRev="b34075b76835b778bb6b2ce0aa224afd9d182887"
-SOURCE_URI="https://github.com/LuaDist/toluapp/archive/$COMMIT_SHA.zip"
+SOURCE_URI="https://github.com/LuaDist/toluapp/archive/$srcGitRev.tar.gz"
-CHECKSUM_SHA256="c69a99a4b0a2bb3decab09bd8d2473911419bef601c86bb6c00ff670f88b4bb1"
+CHECKSUM_SHA256="a86eada2ec3fcc162d2986686eba36e51d095aa471b898d4c641715423310974"
+SOURCE_FILENAME="toluapp-$portVersion-$srcGitRev.tar.gz"
-SOURCE_DIR="toluapp-$COMMIT_SHA"
+SOURCE_DIR="toluapp-$srcGitRev"
```
(Among all our recipes which use a git snapshot, most, if not all, define `srcGitRev`, so we tend to try and stick to that (funny) name :)
(We also prefer using gzipped tarballs instead of ZIP archives.)
Thanks!